Apromore-Custom-Plugins/Process-Discoverer-Portal-Plugin/src/main/java/org/apromore/plugin/portal/processdiscoverer/PDController.java

```diff
@@ -451,6 +434,10 @@ public void openSharingWindow() {
         }
     }
 
+    public void openCalendar() {
+        ((MainController)portalContext.getMainController()).getBaseListboxController().launchCalendar(sourceLogName, sourceLogId);
+    }
+
```

Comment:
As discussed, there is a lack of a communication from CustomCalendar to the hosting plugins like PD. The current issue is after a calendar is changed, e.g. from 27/7 to Monday only, all time measures and statistics in PD still show the values with the old calendar, not the new one.